src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarBattleSessionUtil.java

```diff
@@ -139,6 +132,40 @@ public void run() {
 		}
 	}
 
+	/** 
+	 * Determines how many points are awarded to the winner of a Battle Session.
+	 *
+	 * Affected by the areBattlePointsWinnerTakesAll config setting.
+	 * 
+	 * @param siege the Siege to gather BattlePoints from.
+	 * @return points awarded to the winner.
+	 */
+	public static int calcPointsForBattleSession(Siege siege) {
+		int battlePointsOfWinner;
```

Comment:
:construction: 
- As per previous comments, pls rename to "int siegeBalanceAdjustment"